src/Tests/Unit/Config/MaxLoginConfig.test.ts

```
@@ -208,83 +219,71 @@ describe('MAX_CONFIG', () => {
   describe('possibleResults.success', () => {
-    /**
-     * Extracts the first success predicate from MAX_CONFIG.
-     * @returns The success checker function.
-     */
-    function getSuccessChecker(): (opts: { page?: { url(): string } }) => boolean {
-      return MAX_CONFIG.possibleResults.success[0] as (opts: {
-        page?: { url(): string };
-      }) => boolean;
+    /** First success predicate.
+     * @returns The checker function. */
+    function getChecker(): (o: { page?: { url(): string } }) => boolean {
```

ESLint

[error] 'o' is defined but never used.

(no-unused-vars)